src/core/a11y.js

```diff
@@ -0,0 +1,103 @@
+import { pub } from "./pubsubhub.js";
```

Comment:
Might be nice to have a short pre-amble just stating what the module does and how it works.